apim/3.x/templates/api/api-configmap.yaml

```
@@ -501,7 +501,7 @@ data:
                               class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
                       </jsonFormatter>
                       <appendLineSeparator>true</appendLineSeparator>
-                      <timestampFormat>yyyy-MM-dd' 'HH:mm:ss.SSS</timestampFormat>
+                      <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSXX</timestampFormat>
```

Why not this format ?
```suggestion
                      <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSS'Z'</timestampFormat>
```

---

Because we now that the one we use is working 🙈

---

Maybe @mariano-fernandez @gravitee-io/techops or @gravitee-io/devops know 🤷

---

Ok, forget my comment. I got my answer ;) Z and XX seem equivalent.
With `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'` you will get `2022-08-24T06:12:21.579Z`
But...
With `yyyy-MM-dd'T'HH:mm:ss.SSSZ` you will get `2022-08-24T08:12:21.579+0200
 `
With `yyyy-MM-dd'T'HH:mm:ss.SSSXX` you will get `2022-08-24T08:12:21.579+0200
 `